### PR TITLE
Create ui-c8y-1021-3-0-added-grid-to-the-device-profiles-list.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [DM-3766](https://cumulocity.atlassian.net/browse/DM-3766)
Original PR: [7552](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7552)

[DM-3766]: https://cumulocity.atlassian.net/browse/DM-3766?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1021-3-0-added-grid-to-the-device-profiles-list.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Added grid to the device profiles list.
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1021-3-0-added-grid-to-the-device-profiles-list.md]: The Claude AI suggested to use the following title:
```suggestion
title: Device profiles list now shows a grid
```

File: content/change-logs/device-management/ui-c8y-1021-3-0-added-grid-to-the-device-profiles-list.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3766
+version: 1021.3.0
+---
+Added grid to the device profiles list.
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1021-3-0-added-grid-to-the-device-profiles-list.md]: The Claude AI suggested to use the following body:
```suggestion
To provide a better overview of the device profiles, a grid has been added to the device profiles list in the UI. The grid shows additional properties of the device profiles such as the manufacturer or model which were not visible in the list before. This change makes it easier for users to find and manage specific device profiles without having to open the details of each device profile.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/device-management/ui-c8y-1021-3-0-added-grid-to-the-device-profiles-list.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Device profiles list now shows a grid
```
Comment (dafkk) [inline content/change-logs/device-management/ui-c8y-1021-3-0-added-grid-to-the-device-profiles-list.md]:
```suggestion
title: Device profiles list migrated to grid view
```

File: content/change-logs/device-management/ui-c8y-1021-3-0-added-grid-to-the-device-profiles-list.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Device profiles list migrated to grid view
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1021-3-0-added-grid-to-the-device-profiles-list.md]:
```suggestion
title: Device profiles list now displayed as a grid
```

File: content/change-logs/device-management/ui-c8y-1021-3-0-added-grid-to-the-device-profiles-list.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3766
+version: 1021.3.0
+---
+To provide a better overview of the device profiles, a grid has been added to the device profiles list in the UI. The grid shows additional properties of the device profiles such as the manufacturer or model which were not visible in the list before. This change makes it easier for users to find and manage specific device profiles without having to open the details of each device profile.
```
Comment (dafkk) [inline content/change-logs/device-management/ui-c8y-1021-3-0-added-grid-to-the-device-profiles-list.md]:
```suggestion
The device profiles view has been migrated to a data grid. The data grid provides a more modern and user-friendly way to view and manage device profile items compared to the previous list view. The data grid offers enhanced features such as sorting, filtering, and pagination, making it easier to find and work with specific items.
```

**Review by dafkk — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**